spec/routes/api/project/location/postgres_spec.rb

```diff
@@ -35,7 +35,7 @@
         [:post, "/project/#{project.ubid}/location/#{pg.display_location}/postgres/_#{pg.ubid}/restore"],
         [:post, "/project/#{project.ubid}/location/#{pg.display_location}/postgres/#{pg.name}/reset-superuser-password"],
         [:post, "/project/#{project.ubid}/location/#{pg.display_location}/postgres/_#{pg.ubid}/reset-superuser-password"],
-        [:post, "/project/#{project.ubid}/location/#{pg.display_location}/postgres/_#{pg.ubid}/failover"],
```

Comment:
This should be left for backwards compatibility, unless @fdr thinks we should drop backwards compatibility.  Same for the similar changes in this file.

Reply:
I think we're pre-governance so we could drop it. I think we're still at the stage we could expunge the `_` methodology altogether.

Reply:
Resolving as it seems we are going this direction and it's progressed in #2912 (I'll work on o rebase soon).